view/theme/frio/templates/paginate.tpl

```diff
@@ -0,0 +1,13 @@
+{{if $pager}}

```

Comment:
Maybe I'm too close to the subject, but this template actually is one of the most self-explanatory there are in Friendica, what would you expect as a description?